src/player/Player.php

```diff
@@ -504,6 +508,39 @@ public function isFlying() : bool{
 		return $this->flying;
 	}
 
+	/**
+	 * Sets the player's speed when flying.
+	 *
+	 * The flight speed is calculated as `blocks per tick = flightSpeed * 10`. For example, setting the
+	 * flight speed to 0.05 means the player will fly at approximately 0.5 blocks per tick.
+	 *
+	 * If set to zero, the player will not be able to move in the xz plane when flying, and negative values
+	 * will invert the controls.
+	 *
+	 * Notes:
```

Comment:
Document negative values behavour

Reply:
The negative values behaviour is already documented on the 518 line and at the end of the 517 line, do I need to change something?

Reply:
nah